azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/collection/ResourcesInGroup.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.resources.fluentcore.collection;
+
+public interface ResourcesInGroup<T, DefinitionT> extends
+    SupportsListing<T>,
+    SupportsCreating<DefinitionT>,
+    SupportsDeleting {}
```

Comment:
the collection - as long as it's not InGroup - would have:
`.get(id)` --- perhaps we should even cal it explicitly `.getById()` to distinguish from the other gets
`.getByGroup(groupname, name) -- again, maybe we should use this name here to be more explicit

And the InGroup would have:
`.getByName(name)`